src/rrc_scrape.py

```diff
@@ -0,0 +1,42 @@
+import time
+
+import pandas as pd
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+# this program gets the URLs for the dataset section
```

Comment:
turn this into a function that returns a dataframe.
 the dataframe will have the filetype, e.g. Ascii, EBCDOC, ArcFile
the uri will be the URL to the list of files
the dataset_description will be the description from the main RRC dataset page.